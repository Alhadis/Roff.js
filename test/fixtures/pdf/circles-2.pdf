%PDF-1.4
%‚„œ”
3 0 obj << /Contents [4 0 R  ]
/Group << /CS /DeviceRGB
/S /Transparency
>>
/Parent 2 0 R
/Type /Page
>>
endobj
4 0 obj << /Length 2131
>>
stream
q 1 0 0 1 0 0 cm
1 J
1 j
% V12000
%% H72000
% DFd
0 g
% h54000
% n12000 0
q BT
1 0 0 1 126 780 Tm

0 Tc
% V12000
%% H72000
% n12000 0
1 0 0 1 72 780 Tm
0 Tc
% V12000
%% H72000
% DFg 0
1 0 0 1 72 780 Tm
0 Tc
ET Q
0 g
% h18000
% md
0 G
% s10000
% DP 7200 -1800 0 3600
90 780 m
97.2 781.8 l
97.2 778.2 l
f
% n12000 0
q BT
1 0 0 1 97.2 778.2 Tm
0 Tc
% V12000
%% H72000
% Dt 100 0
1 0 0 1 72 780 Tm
0 Tc
ET Q
0.1 w
% H72000
% n12000 0
q BT
1 0 0 1 72 780 Tm
0 Tc
% V12000
%% H90000
% Dp 7200 -1800 0 3600
1 0 0 1 90 780 Tm
0 Tc
ET Q
90 780 m
97.2 781.8 l
97.2 778.2 l
s
% n12000 0
q BT
1 0 0 1 97.2 778.2 Tm
0 Tc
% V12000
%% H72000
% Dt -1000 0
1 0 0 1 72 780 Tm
0 Tc
ET Q
0.4 w
% h1000
% n12000 0
q BT
1 0 0 1 73.4 780 Tm
0 Tc
% V12000
%% H126000
% Dl -28800 0
1 0 0 1 126 780 Tm
0 Tc
ET Q
126 780 m
97.2 780 l
s
% n12000 0
q BT
1 0 0 1 97.2 780 Tm
0 Tc
% V12000
%% H90000
% Da 0 18000 -18000 0
1 0 0 1 90 780 Tm
0 Tc
ET Q
q 18 0 0 18 90 762 cm -0.195090322016128 0.98078528040323 -0.98078528040323 -0.195090322016128 0 0 cm 0.0222222222222222 w 0.98078528040323 0.195090322016128 m 1.00640490653226 0.0662917638041763 1.00640490653226 -0.0662917638041763 0.98078528040323 -0.195090322016128 c S Q
q 18 0 0 18 90 762 cm -0.555570233019602 0.831469612302545 -0.831469612302545 -0.555570233019602 0 0 cm 0.0222222222222222 w 0.98078528040323 0.195090322016128 m 1.00640490653226 0.0662917638041763 1.00640490653226 -0.0662917638041763 0.98078528040323 -0.195090322016128 c S Q
q 18 0 0 18 90 762 cm -0.831469612302545 0.555570233019602 -0.555570233019602 -0.831469612302545 0 0 cm 0.0222222222222222 w 0.98078528040323 0.195090322016128 m 1.00640490653226 0.0662917638041763 1.00640490653226 -0.0662917638041763 0.98078528040323 -0.195090322016128 c S Q
q 18 0 0 18 90 762 cm -0.98078528040323 0.195090322016129 -0.195090322016129 -0.98078528040323 0 0 cm 0.0222222222222222 w 0.98078528040323 0.195090322016128 m 1.00640490653226 0.0662917638041763 1.00640490653226 -0.0662917638041763 0.98078528040323 -0.195090322016128 c S Q
% n12000 0
q BT
1 0 0 1 72 762 Tm
0 Tc
% x trailer
% V792000
%% x stop
% x stop
1 0 0 1 72 0 Tm
0 Tc
ET Q
Q
endstream
endobj
1 0 obj << /Pages 2 0 R 
/Type /Catalog
>>
endobj
5 0 obj << /CreationDate (D:20170704152946+10'00')
/Creator (groff version 1.22.3)
/ModDate (D:20170704152946+10'00')
/Producer (gropdf version 1.22.3)
>>
endobj
2 0 obj << /Count 1
/Kids [3 0 R  ]
/MediaBox [0 0 612 792 ]
/Resources << /Font << >>
/ProcSet [/PDF /Text /ImageB /ImageC /ImageI ]
>>
/Rotate 0
/Type /Pages
>>
endobj
xref
0 6
0000000000 65535 f 
0000002309 00000 n 
0000002521 00000 n 
0000000015 00000 n 
0000000127 00000 n 
0000002359 00000 n 
trailer
<<
/Info 5 0 R 
/Root 1 0 R
/Size 6
>>
startxref
2691
%%EOF
