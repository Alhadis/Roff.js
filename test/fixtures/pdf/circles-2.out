x T pdf
x res 72000 1 1
x init
x F fixtures/circles-2.pic
p1
V12000
H72000
DFd
h54000
n12000 0
V12000
H72000
n12000 0
V12000
H72000
DFg 0
h18000
md
s10000
DP 7200 -1800 0 3600
n12000 0
V12000
H72000
Dt 100 0
H72000
n12000 0
V12000
H90000
Dp 7200 -1800 0 3600
n12000 0
V12000
H72000
Dt -1000 0
h1000
n12000 0
V12000
H126000
Dl -28800 0
n12000 0
V12000
H90000
Da 0 18000 -18000 0
n12000 0
x trailer
V792000
x stop
